sympy/stats/tests/test_joint_rv.py

```diff
@@ -53,6 +53,32 @@ def test_NormalGamma():
         3*sqrt(10)*gamma(S(7)/4)/(10*sqrt(pi)*gamma(S(5)/4))
     assert marginal_distribution(ng, y)(1) == exp(-S(1)/4)/128
 
+def test_MultivariateBeta():
+    from sympy.stats.joint_rv_types import MultivariateBeta
+    from sympy import gamma
+    a1, a2 = symbols('a1, a2', positive=True)
+    a1_f, a2_f = symbols('a1, a2', positive=False)
+    mb = MultivariateBeta('B', [a1, a2])
+    mb_c = MultivariateBeta('C', a1, a2)
+    assert simplify(density(mb)(1, 2) -
```

Comment:
Can `simplify` be avoided here (and similarly below)?

Reply:
I used `simplify` as it was in the FAQ, [Why does SymPy say that two equal expressions are unequal?](https://github.com/sympy/sympy/wiki/Faq#why-does-sympy-say-that-two-equal-expressions-are-unequal).

Reply:
It can be slow as it tries numerous simplifications. It is better if the output can be compared directly without having to using `simplify`.

Reply:
Since, this is just some sympy code at work, I have removed `simplify`. If there will be any problem, we can handle it at that time, for now the performance of the code matters.